schema.ts

```diff
@@ -568,6 +568,40 @@ export interface StringDateTimeSchema extends BaseStringSchema<ValueHintType.Dat
   timeFormat?: string;
 }
 
+/**
+ * State of outline on images that can be used with a {@link ImageSchema}.
+ */
+export enum Outline {
+  /** Image outline is disabled. */
+  Disabled = 'disabled',
+  /** Image outline is applied. */
+  Default = 'default',
+}
+
+/**
+ * State of rounded corners on images that can be used with a {@link ImageSchema}.
+ */
+export enum RoundedCorners {
+  /** Image rounded corners are disabled. */
+  Disabled = 'disabled',
+  /** Image rounded corners are applied. */
+  Default = 'default',
+}
+
+/**
+ * A schema representing a return value or object property that is provided as a string,
+ * which Coda should interpret as an image.
+ */
+ export interface ImageSchema extends BaseStringSchema<
+  ValueHintType.ImageReference | ValueHintType.ImageAttachment> {
+  /** Instructs Coda to render this value as an Image. */
+  codaType: ValueHintType.ImageReference | ValueHintType.ImageAttachment;
+  /** Outline type specifying whether or not to add outline to rendered images. Defaults to true. */
```

Comment:
> Defaults to true.

This isn't a boolean field, so not quite accurate. Either change to an enum, or perhaps just leave unspecified, since it's likely to change in the product without being updated here.

Ditto below.